### PR TITLE
fix: Stop processing infinite idms copybooks loop

### PR DESCRIPTION
**Pull request by Nurkambay**

### CODE REVIEW

File: server/src/main/resources/resourceBundles/messages_en.properties [hunk 1]
```diff
@@ -85,4 +85,6 @@ variables.integer=integer
 variables.blankWhenZero=BLANK WHEN ZERO
 variables.signClause=SIGN CLAUSE
 variables.mapName=Map name
+IdmsCopybookVisitor.errorCircularDependency=%s: Copybook has circular dependency
```
Comment (Nurkambay) [inline server/src/main/resources/resourceBundles/messages_en.properties:88]: @zeibura can you help with the text?

File: server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/ErrorHelper.java [hunk 2]
```diff
@@ -44,4 +44,19 @@ public SyntaxError reportMissingCopybooks(MessageService messageService, Localit
     LOG.debug("Syntax error by reportMissingCopybooks: {}", error.toString());
     return error;
   }
+
+  public SyntaxError reportCircularDependency(MessageService messageService, Locality locality, String copybookName) {
```
Comment (ishche) [inline server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/ErrorHelper.java:48]: How do you feel about dropping "report" from method names?

Comment (Nurkambay) [reply server/src/main/java/org/eclipse/lsp/cobol/core/engine/dialects/idms/ErrorHelper.java:48]: I am okay with that!

**Review by ishche — APPROVED** [review]

**Review by zeibura — APPROVED** [review]

File: server/src/main/resources/resourceBundles/messages_en.properties (hunk 1 above)
Comment (zeibura) [reply server/src/main/resources/resourceBundles/messages_en.properties:88]: Maybe change it to "Copybook has circular dependencies" but its fine really.

**Pull request merged**